README.md

```diff
@@ -48,6 +49,36 @@ curl --request POST \
 
 5) Open [http://localhost:8080](http://localhost:8080) to see your uploaded docs !
 
+### Delete a doc
+
+1) Launch the application and its server with ```docker run -e CREDENTIALS=user:password -e SHOULD_SECURE=FALSE -v `pwd`:/data -p 8080:80 tracesoftware/hostmydocs```
+
+2) delete using cURL by example :
+```bash
+curl --request DELETE \
+    --url http://localhost:8080/BackEnd/deleteProject \
+    --user user:password \
+    -d "name=DocumentationName" \
+    -d "version=1.0.0" \
+    -d "language=YourProgrammingLanguage"
+```
+
+> Note that you can write
```

Comment:
Why using a citation here ?

Reply:
I see citation in README as a way to give optionals informations, how would you do that ?